UnityProject/Assets/Scripts/Systems/Disposals/DisposalTraversal.cs

```diff
@@ -291,7 +294,10 @@ private void EjectViaDisposalMachine(DisposalMachine machine)
 		private void TryDamageTileFromEjection(Vector3Int localPosition)
 		{
 			if (matrix.TileChangeManager.MetaTileMap.HasTile(localPosition, LayerType.Floors) == false) return;
-			matrix.TileChangeManager.MetaTileMap.SetTile(localPosition, TileType.Floor, "damaged3");
+			matrix.TileChangeManager.MetaTileMap.ApplyDamage(
```

Comment:
the only issue is it will eventually punch a hole to space

Reply:
It wont, it only affects the upper floor layers. not the under ones.